client/public/index.html

```diff
@@ -2,28 +2,24 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <!-- font link -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Shippori+Mincho:wght@500&display=swap" rel="stylesheet">
+<!-- BOB - all the TA'S LOOKED AT THIS CARROT. if you get rid of it.... the markers image does not show. I have no idea, neither does anyone else. that is why we have a carrot in the top corner BUT if its not there.... the marker images glitch out.  -->
+    <
```

Comment:
Edit: I'm still not 100% sure what the deal is with this one now, it looks like this was part of the repo code at one point possibly.. hopefully Bob can help us out with this one. I wonder if it is something to do with the closing tags on the two links above and how React is correcting that in the browser?
![Screenshot_2021-03-09 Feedback by github-classroom · Pull Request #1 · BurlingtonCodeAcademy yelpington-orobb33(2)](https://user-images.githubusercontent.com/66134436/110533298-d2f6ed80-80eb-11eb-9a29-785d6aea6617.png)